modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/BasicInformation.js

```diff
@@ -51,47 +52,41 @@ function BasicInformation({author, canonicalURL, publishDate, title}) {
 	return (
 		<div className="sidebar-section">
 			<ClayLayout.ContentRow>
-				<ClayLayout.ContentCol expand>
+				<ClayTooltipProvider>
 					<span
 						className="component-title text-truncate-inline"
-						data-tooltip-align="top"
+						data-tooltip-align="bottom"
 						title={title}
 					>
 						<span className="text-truncate">{title}</span>
 					</span>
-				</ClayLayout.ContentCol>
+				</ClayTooltipProvider>
 			</ClayLayout.ContentRow>
 
 			<ClayLayout.ContentRow>
-				<ClayLayout.ContentCol expand>
+				<ClayTooltipProvider>
 					<span
-						className="text-truncate-inline"
-						data-tooltip-align="top"
+						className="c-mb-2 c-mt-1 text-truncate text-truncate-reverse"
+						data-tooltip-align="bottom"
 						title={canonicalURL}
 					>
-						<span className="c-mb-2 c-mt-1 text-secondary text-truncate text-truncate-reverse">
-							{canonicalURL}
-						</span>
+						<bdi className="text-secondary">{canonicalURL}</bdi>
 					</span>
-				</ClayLayout.ContentCol>
+				</ClayTooltipProvider>
```

Comment:
> Those were missing to show the Tooltip as the clay one instead of the plain browser one 😓 I fix that too

Is it possible to hoist the provider up instead of repeating it? I am not sure why you need it at all because in theory anything rendered by `<react:component>` gets one "for free", but we have had some reports of tool-tip bugs of late (not sure what the current status is there)... notwithstanding those bugs, I don'd know why you would need multiple providers at this level (see [the Storybook example](https://storybook.clayui.com/?path=/story/components-claytooltip--tooltip), which shows multiple items having tooltips supported by a single provider at the top).

Reply:
> not sure what the current status is there

I was taking a quick look yesterday and it indeeds does not seem to be working but couldn't dedicate enough time to figure out why. The fact that the module `tooltip-support-web` covers some cases within react components adds to the confusion because it seems like the problem was with portals and it turns out it isn't 🤔 

The main context is being gathered at [ClayTooltip not properly displayed in Page Editor Sidebar](https://issues.liferay.com/browse/LPS-129309) where we've asked other teams to report their findings instead of applying individual fixes so we can analyze in detail in the upcoming weeks.

> notwithstanding those bugs, I don'd know why you would need multiple providers at this level (see the Storybook example, which shows multiple items having tooltips supported by a single provider at the top).

That should be the case, yeah.

Reply:
Ok, I'll remove the ClayTooltipProvider fix within this pull and report the issue about it 👍  Seems reasonable to me. Thanks guys!

Reply:
Here I put our case with an image attached: https://issues.liferay.com/browse/LPS-129309?focusedCommentId=2405163&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-2405163. Thanks!